### PR TITLE
Updated Readme and contributing md files

### LINKED ISSUE #42: Update documentation
[README.md](https://github.com/laurenball/Steven/blob/master/README.md) is out of date: 
  - include missing commands
    - [ ] !removeaction - This is **owner only** command
    - [ ] !help
    - [ ] !display
  - include missing event triggers (in the 'hooks' section of the readme)
    - [ ] mentions

[CONTRIBUTING.md](https://github.com/laurenball/Steven/blob/master/CONTRIBUTING.md) is out of date:
  - [ ] New branches should be created from `develop`
  - [ ] Ready to review branches should be up to date with `develop`
  - [ ] Remove label part step

### PR DESCRIPTION
**Pull request by viveksoundrapandi**
Updated the files as per the open issue

fixes #42 

Let me know if any changes. Happy to help

### CODE REVIEW

**Comment by cam-ball** [comment]
@viveksoundrapandi Thanks for the PR! The issue related to this has already been assigned to someone else, so I'd like to give them time to make the changes they were going to make, but if they don't come back in the next week or so, I can move forward with formally reviewing your PR.

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ Discord bot for morale improvement.
 #### Hooks
 - User-specific interaction:
   - Follows owner-selected users and occasionally showers them with praise
+  - Mentions
```
Comment (cam-ball) [inline README.md]: @viveksoundrapandi could you move this into the `general interaction` section and maybe update the wording so it's a full sentence?

**Review by cam-ball — APPROVED** [review]
Just one quick tweak and this is good to go. Thank you!

File: README.md (hunk 1 above)
Comment (viveksoundrapandi) [reply README.md]: @laurenball sure

Comment (viveksoundrapandi) [reply README.md]: moved mentions to general interaction with a gist message

**Comment by cam-ball** [comment]
thanks @viveksoundrapandi

**Pull request merged**